src/avram/query_builder.cr

```diff
@@ -298,17 +298,35 @@ class Avram::QueryBuilder
     where(where_clause)
   end
 
+  def or(&block : Avram::QueryBuilder -> Avram::QueryBuilder)
+    if @wheres.empty?
+      raise Avram::InvalidQueryError.new("Cannot call `or` before calling a `where`")
+    end
+
+    @wheres.last.conjunction = Avram::Where::Conjunction::Or
+
+    block.call(self)
+    self
+  end
+
   @_wheres_sql : String?
 
   private def wheres_sql
     @_wheres_sql ||= joined_wheres_queries
   end
 
   private def joined_wheres_queries
-    if wheres.any?
-      statements = wheres.map(&.prepare(->next_prepared_statement_placeholder))
+    if @wheres.any?
```

Comment:
this should actually call the method instead of instance variable, but I was trying to eliminate the `uniq` in case that's what was causing the spec to fail. That doesn't seem to be the case though....

Reply:
The two uses of `@wheres` in here are causing one of the specs to fail. Once this and the other changes are made, all specs should be passing.